apiserver/clusterservice/clusterimpl.go

```diff
@@ -1133,6 +1133,16 @@ func getClusterParams(request *msgs.CreateClusterRequest, name string, userLabel
 	} else {
 		spec.CCPImage = "crunchy-postgres-ha"
 	}
+
+	// update the CRD spec to use the custom CCPImagePrefix, if given
+	// otherwise, set the value from the global configuration
+	if request.CCPImagePrefix != "" {
+		spec.CCPImagePrefix = request.CCPImagePrefix
+		log.Debugf("user is overriding CCPImagePrefix from command line %s", request.CCPImagePrefix)
+	} else {
+		spec.CCPImagePrefix = apiserver.Pgo.Cluster.CCPImagePrefix
+	}
```

Comment:
Why not user that operator `GetImagePrefix` function here?

Or make the `GetImagePrefix` function a utility function used across apiserver/operator?

Not a requirement, but seems like a way to not duplicate logic.

Reply:
...adding on to this thought, based on how `GetImagePrefix` works, do we even need to have the `else` block here? That function automatically sets the image prefix.

We should also have a fallback if neither the user specifies it one off in `pgo create cluster` nor the `pgo.yaml` setting is set. I would propose that we have a `const DefaultImagePrefix` set in the cluster.go CRD file that just sets it to `crunchydata`

Reply:
My initial thought was not to use `GetImagePrefix` here because it will require importing the `operator` package where it hasn't been previously. Is that worth using the function here? If so, happy to make the change.